student.rb

```diff
@@ -8,6 +8,11 @@ def initialize(classroom, age, name = 'Unknown', id = Random.rand(1..1000), pare
     @classroom = classroom
   end
 
+  def classroom=(classroom)
+    @classroom = classroom
+    classroom.student << self
+  end
+
```

Comment:
- [ ] Your method should contain an **unless** or **if statement** for the exception of if the student exists in the classroom or not :thinking:

```ruby
  def classroom=(classroom)
      @classroom = classroom
      classroom.students.push(self) unless classroom.students.include?(self)
  end
```